NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -461,6 +470,7 @@ class NeuroID private constructor(
 
     fun stop() {
         pauseCollection()
+        nidCallActivityListener?.unregisterCallActivityListener(this.getApplicationContext())
```

Comment:
we remove this on stop/stopSession, are we re-registering when they call start? They won't call configure more than 1 time

Reply:
added to start and startSession that registers if not already.